docarray/array/storage/weaviate/getsetdel.py

```diff
@@ -77,8 +77,12 @@ def _clear_storage(self):
             self._load_or_create_weaviate_schema()
 
     def _load_offset2ids(self):
-        ids, self._offset2ids_wid = self._get_offset2ids_meta()
-        self._offset2ids = Offset2ID(ids)
+        if self._list_like:
+            ids, self._offset2ids_wid = self._get_offset2ids_meta()
+            self._offset2ids = Offset2ID([], list_like=self._list_like)
```

Comment:
it seems weird that it is an empty list? why are tests not failing

Reply:
Because when list_like is False, self._offset2id will not do anything else, just not to make existing changes.